ext/nnstreamer/tensor_decoder/tensordec-pose.c

```diff
@@ -513,6 +513,25 @@ typedef struct
   gfloat prob;
 } pose;
 
+/**
+ * @Brief Check if a value is within lower and upper bounds
```

Comment:
"@Brief" -> "@brief" to relieve CI's complain.
You may also consider to sign-off your commit and add some commit body (other than commit head).